keycloak/keycloak-admin-ui
# Realm settings(keys): Adds keys list to Realm settings keys tab

## Motivation

https://issues.redhat.com/browse/APPDUX-934

## Verification Steps


1. Go to Realm settings -> Keys tab in the old admin console
2. Navigate to the "Providers " subtab and click "Add keystore..." in the top right of the data table.
2. Select a keystore type and fill out the form.
3. Click save and go to Realm settings -> Keys tab in the new admin console
4. Verify that the data listed in the table matches the design, and the data listed in the old console

## Checklist:
